src/assets/BookList.css

```diff
@@ -1,43 +1,200 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab:wght@700&display=swap');
+
```

Comment:
- [ ] Your app looks awesome. You made your app styling exactly the same as provided on Zeplin. Good job! However, when running the npm start I am having an error 👇 
![Screenshot (207)](https://user-images.githubusercontent.com/67087664/155705514-026cec1e-eecb-4138-a4d6-3f4d2db1db3d.png)
Kindly fix it before requesting for another review 🙏

Reply:
resolved

Reply:
Fixed :+1: